tests/unit-tests/test_model_save_load.py

```diff
@@ -85,17 +86,25 @@ def test_sparse_embed_save(world_size):
         sparse_embs = \
             {ntype: sparse_emb._tensor[th.arange(embed_layer.g.number_of_nodes(ntype))] \
                 for ntype, sparse_emb in embed_layer.sparse_embeds.items()}
-        for i in range(world_size):
-            save_sparse_embeds(model_path, embed_layer, i, world_size)
 
-        for ntype in embed_layer.sparse_embeds.keys():
-            saved_embs = []
+        @patch("graphstorm.model.utils.get_rank")
+        @patch("graphstorm.model.utils.get_world_size")
```

Comment:
what does this do?

Reply:
mock get_rank and get_world_size to mimic different output.